lib/Page.js

```diff
@@ -511,12 +511,29 @@ class Page extends EventEmitter {
    */
   async _onBindingCalled(event) {
     const {name, seq, args} = JSON.parse(event.payload);
-    const result = await this._pageBindings.get(name)(...args);
-    const expression = helper.evaluationString(deliverResult, name, seq, result);
+    let result = null;
+    let error = null;
+    try {
+      result = await this._pageBindings.get(name)(...args);
+    } catch (e) {
+      error = e;
+    }
+    let expression = null;
+    if (error)
```

Comment:
Things can throw null. Instead of checking if there is an error, do the logic in the catch.

Reply:
Fair. Creepy, but fair.